python_modules/libraries/dagster-airflow/dagster_airflow/operators/operator_to_op.py

```diff
@@ -0,0 +1,53 @@
+import logging
+import os
+from contextlib import contextmanager
+from typing import List, Optional
+
+from airflow.models import Connection
+from dagster import check, op, Any, In, Nothing, OpDefinition, Out
+
+
+@contextmanager
+def replace_airflow_logger_handlers(handler):
+    try:
+        root_logger = logging.getLogger()
+        root_logger.addHandler(handler)
+        yield
+    finally:
+        root_logger.removeHandler(handler)
+
+
+def operator_to_op(
+    airflow_op,
+    connections: Optional[List[Connection]] = None,
+    capture_python_logs=True,
+    return_output=False,
+) -> OpDefinition:
+    connections = check.opt_list_param(connections, "connections", Connection)
+
+    for connection in connections:
```

Comment:
I think this code should be inside the op. right now, it will set these environment variables on whatever machine that the job is defined in, but that might not be the same place that the op actually runs. we might also want to unset these variables once the op completes, just to keep things clean.

Reply:
This makes a lot of sense. Thanks, adding this